lib/gds_api/support_api.rb

```diff
@@ -21,6 +21,26 @@ def create_global_export_request(request_details)
     post_json!("#{endpoint}/anonymous-feedback/global-export-requests", global_export_request: request_details)
   end
 
+  # Create a Page Improvement
+  #
+  # Makes a +POST+ request to the support api to create a Page Improvement.
+  #
+  # @param params [Hash] Any attributes that relate to a Page Improvement.
```

Comment:
YARD seems to accept the type before the name of the attribute (`@param [Hash] params description` - http://www.yardoc.org/features.html)